components/common/ConnectWallet/ConnectWalletModal.vue

```diff
@@ -96,15 +148,16 @@ const setAccount = (account: Auth) => {
 }
 
 const installedWallet = computed(() => {
-  return wallets.filter((wallet) => wallet.installed)
+  return wallets.value.filter((wallet) => wallet.installed)
 })
 const uninstalledWallet = computed(() => {
-  return wallets.filter((wallet) => !wallet.installed)
+  return wallets.value.filter((wallet) => !wallet.installed)
 })
-const showUninstalledWallet = ref(!installedWallet.value.length)
+const showUninstalledWallet = computed(() => !installedWallet.value.length)
 
+const isUninstalledWalletExpand = ref(!showUninstalledWallet.value)
```

Comment:
shoudn't this be a `computed`?

Reply:
No. `isUninstalledWalletExpand` should be `ref` because it's used to control the direction of the dropdown icon